packages/specs/l2geth/sequencer/README.md

```diff
@@ -0,0 +1,109 @@
+# Sequencer specs
```

Comment:
`/seqencer` had it's own top level dir in [the specs repo](https://github.com/ethereum-optimism/specs/tree/main/sequencer). I moved it into `/l2geth` here. I supposed I could have done: 
```
mv l2geth/sequencer/* l2geth
rm -rf sequencer
```

But this felt like a more conservative approach to content written by others.